docs/en/compute/Presto-Iceberg.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
+nickname: Presto-Iceberg
+group: Compute Integrations
+priority: 2
+---
+
+Presto has introduced support for [Iceberg tables](https://iceberg.apache.org/) in version 0.256.
+
+This document describes how to use Presto to query Iceberg tables through Alluxio.
+
+* Table of Contents
+{:toc}
+
+In order to use Presto to query an Iceberg table, make sure you have a working setup of Presto, 
+Hive Metastore and Alluxio, and Presto can access data through Alluxio's filesystem interface.
+If not, please refer to the 
+[guide]({{ '/en/compute/Presto.html' | relativize_url }}) on general Presto installation 
+and configuration. Most of that guide apply for Iceberg workflows as well, and this document 
+covers the specific instructions for working with Iceberg tables.
+
+The [guide]({{ '/en/compute/Presto.html' | relativize_url }}) mentions two approaches that 
+Presto can access data through Alluxio: via the filesystem interface, and via the catalog service. 
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
+needs to access them through Alluxio's filesystem interface.
+
+## Install Alluxio client jar to Presto Iceberg connector
```

Comment:
Let's have the outline like this (similar as Presto page):

<pre>

## Prerequisites
Put what you require

## Basic Setup
Describe the bare minimal setup to let users run this stack

## Examples
Put the query example

## Advanced Setup
Any advanced configuration (non default)
</pre>